web_src/js/components/DiffFileTreeItem.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <!-- eslint-disable -->
```

Comment:
What's the problem?

Reply:
this was just a copy/paste thing from a different component... removed it.